### PR TITLE
Core Supermatter

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

**Walks into a room**
**Slaps down CORE, but with Supermatter**
**Doesn't elaborate**
**Leaves**

<details><summary><h1>Media</h1></summary>
<p>

Core-0

image

</p>
</details>

# Changelog

:cl:
- add: Core has been added to the list of maps in rotation. This time featuring a custom Supermatter Engine

### CODE REVIEW

File: Resources/Prototypes/Maps/core.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            # Logistics
+            CargoTechnician: [ 3, 3 ]
+            Quartermaster: [ 1, 1 ]
+            SalvageSpecialist: [ 2, 2 ]
+            # Civilian
+            Clown: [ 1, 1 ]
+            Mime: [ 1, 1 ]
+            Musician: [ 1, 1 ]
+            Boxer: [ 2, 2 ]
+            Reporter: [ 2, 2 ]
```
Comment (DangerRevolution) [inline Resources/Prototypes/Maps/core.yml:66]: I know this is the same config as Wizden's Core but this is stupid, starts at 35 and doesn't open any more job slots even if it grows to 75 or higher.

Some of these should have some spare Latejoin slots to account for people not cryoing and SSDing or just general pop growth

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Maps/core.yml (hunk 1 above)
Comment (VMSolidus) [reply Resources/Prototypes/Maps/core.yml:66]: I uhhh... Actually don't normally mess with files like this, so I have no idea how the job slot things work lol. Are you saying the 2nd number is additional slots that open up as the pop increases?

Comment (DangerRevolution) [reply Resources/Prototypes/Maps/core.yml:66]: Kinda,
first number is round-start.
second number is latejoin slots
-1 is infinite

Comment (VMSolidus) [reply Resources/Prototypes/Maps/core.yml:66]: Well, you probably know this a lot better than me. Do you mind suggesting what the numbers would be better off being?

**Review by Peptide90 — APPROVED** [review]

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**